pkgs/tools/networking/connman/default.nix

```
@@ -52,6 +52,8 @@
 assert stdenv.lib.asserts.assertOneOf "firewallType" firewallType [ "iptables" "nftables" ];
 assert stdenv.lib.asserts.assertOneOf "dnsType" dnsType [ "internal" "systemd-resolved" ];
 
+let inherit (stdenv.lib) optionals; 
+
```

This is currently broken
```suggestion
let 
  inherit (stdenv.lib) optionals;
in
```

---

Right, I haven't tested that. I got it fixed in last push.